database/queries.sql

```diff
@@ -0,0 +1,118 @@
+-- psql -d qa -f database/queries.sql
```

Comment:
Looks like this file is not hidden as you expected :)

Reply:
I have another file called `queries.sql` in the root directory too. That's the one being ignored.